.travis.yml

```diff
@@ -42,4 +42,4 @@ jobs:
         github_token: $TRAVIS_GH_TOKEN
         local_dir: dist/GreenCityClient
         on:
-          branch: dev
+          branch: dev
```

Comment:
Please, fix travis error

Reply:
i dont understand where the travis error is